src/index.js

```diff
@@ -1604,6 +1630,30 @@ const initializeFormElements = () => {
     console.log(result);
   };
 
+  // Malicious Mint
+  maliciousContractInteractionButton.onclick = async () => {
+    let contractAddress;
+
+    if (networkName === 'mainnet') {
+      contractAddress = '0x000062Accd1a9d62eF428eC86cA3dD4f45120000';
+    } else {
+      contractAddress = '0x00008F1149168C1D2fa1eBa1Ad3e9cD644510000';
+    }
+
+    const result = await provider.request({
+      method: 'eth_sendTransaction',
+      params: [
+        {
+          from: accounts[0],
+          to: contractAddress,
+          data: '0xef5cfb8c0000000000000000000000000b3e87a076ac4b0d1975f0f232444af6deb96c59',
```

Comment:
interesting! this calldata is calling the function `claimRewards` but that doesn't exist in the contracts I've checked. However I do see the Blockaid malicious warning, so I guess it's fine, even if that would make it revert

Reply:
the payload and the accounts are what blockaid are using to test it, so I guess it is ok indeed